workspaces/bulk-import/plugins/bulk-import/src/components/Repositories/AddedRepositoryTableRow.tsx

```diff
@@ -71,16 +78,26 @@ export const AddedRepositoryTableRow = ({
   data: AddRepositoryData;
 }) => {
   const classes = useStyles();
-
+  const importFlow = useImportFlow();
   return (
     <TableRow hover>
       <TableCell component="th" scope="row" className={classes.tableCellStyle}>
-        {data.repoName}
+        {importFlow === 'scaffolder' ? (
+          <Link
+            to={`/bulk-import/repositories/tasks/${encodeURIComponent(
+              data.repoUrl || '',
+            )}`}
+          >
+            {data.repoName}
+          </Link>
+        ) : (
+          data.repoName
+        )}
       </TableCell>
       <TableCell align="left" className={classes.tableCellStyle}>
         {data?.repoUrl ? (
-          <Link to={data.repoUrl}>
-            {urlHelper(data.repoUrl)}
+          <Link to={data.repoUrl || ''}>
+            {urlHelper(data.repoUrl || '')}
```

Comment:
This is maybe introduced to a merge conflict? But we don't need this fallback here because there is a  condition around this above.